1-exercises/A-undefined/exercise.js

```diff
@@ -12,25 +12,26 @@
 // Example 1
 let a;
 console.log(a);
-
+/* a is not defined */
 
 // Example 2
 function sayHello() {
     let message = "Hello";
 }
-
+/* there is no return */
 let hello = sayHello();
 console.log(hello);
-
+/* hello is a variable not a function */
 
 // Example 3
 function sayHelloToUser(user) {
     console.log(`Hello ${user}`);
 }
 
 sayHelloToUser();
-
+/*needs a parameter*/
```

Comment:
technically, it's an `argument` (parameter is `user`, argument is the value assigned to `user` when invoking a function)